client/fresheyes-web/src/app/components/form/proceess-pr.ts

```diff
@@ -0,0 +1,42 @@
+"use server";
+
+import { auth } from "@/auth";
+import { API_URL } from "@/config/process";
+import { PullRequest, PullRequestResponse } from "@/types";
+
+export const processPr = async (args: PullRequest) => {
+  const { owner, repo, pull_number } = args;
+  const session = await auth();
+  if (!session || !session?.accessToken) {
+    return {
+      error: "You must be logged in to process a pull request",
+      data: null,
+    };
+  }
+
+  const response = await fetch(`${API_URL}/process_pull_request`, {
```

Comment:
Non blocking but is this how the path is defined with underscore and not dashes

Reply:
Yes. Thats the endpoint from the server